frontend/packages/dev-console/src/components/health-checks/AddHealthChecks.tsx

```diff
@@ -50,6 +49,10 @@ const AddHealthChecks: React.FC<FormikProps<FormikValues> & AddHealthChecksProps
   } = resource;
   const kindForCRDResource = referenceFor(resource);
   const resourceKind = modelFor(kindForCRDResource).crd ? kindForCRDResource : kind;
+  const {
+    values: { healthChecks },
+  } = useFormikContext<FormikValues>();
```

Comment:
No need to get the value from using the hook as ```values``` is already present in this component just destructure the values from the props and use it.

Reply:
Updated.